tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -374,3 +374,10 @@ test_spi_adt7310: adt7310@2d {
 	spi-max-frequency = <0>;
 	int-gpios = <&test_gpio 0 0>;
 };
+
+test_spi_bmi323: bmi323@2b {
```

Comment:
Update the address to be incremented by one from the last entry

Reply:
Updated to use address which is incremented by one from last entry :)